source/PyMaterialX/PyMaterialXGenShader/PyUnitConverter.cpp

```diff
@@ -84,7 +84,7 @@ void bindPyUnitConverters(py::module& mod)
         .def("getUnitAsInteger", &mx::LinearUnitConverter::getUnitAsInteger)
         .def("getUnitFromInteger", &mx::LinearUnitConverter::getUnitFromInteger);
 
-    py::class_<mx::UnitConverterRegistry, std::unique_ptr<mx::UnitConverterRegistry, py::nodelete>>(mod, "UnitConverterRegistry")
+    py::class_<mx::UnitConverterRegistry, mx::UnitConverterRegistryPtr>(mod, "UnitConverterRegistry")
         .def_static("create", &mx::UnitConverterRegistry::create)
```

Comment:
Makes units usable under Python.
The functions `UnitConverterRegistry::create()`, `UnitSystem::setUnitConverterRegistry()` and `UnitSystem::getUnitConverterRegistry()` expects shared_ptr, not unique_ptr.

Reply:
Thanks for fixing this. I think the signature used to be unique_ptr.